### ISSUE TITLE
Notification adding to conda-forge

### ISSUE THREAD

**Issue by jfy133** [hsolbrig/ShExJSG#11]
I would like to let you know I am adding ShExJSG to the conda-forge code distribution platform (in parallel to your existing pypi package), in the context of the LinkML project.

For your reference, the conda-forge PR is here: https://github.com/conda-forge/staged-recipes/pull/26816

**Comment by jfy133** [comment]
It is now merged into conda-forge
